doc/build-osx.md

```diff
@@ -96,11 +96,14 @@ Skip if you don't intend to use the GUI.
 brew install qt@5
 ```
 
-Ensure that the `qt@5` package is installed, not the `qt` package.
-If 'qt' is installed, the build process will fail.
-if installed, remove the `qt` package with the following command:
+Note: Ensure that _only_ `qt@5` is installed:
+
```

Comment:
nit: `brew list --version | grep qt` is the command to ensure that only qt@5 is installed, so maybe we could move it just after the CTA?
```suggestion
Note: Ensure that _only_ `qt@5` is installed:
``` bash
brew list --version | grep qt
```
```